lib/abci/handlers/deliverTxHandlerFactory.js

```diff
@@ -29,8 +40,12 @@ function deliverTxHandlerFactory(
    * @return {Promise<abci.ResponseDeliverTx>}
    */
   async function deliverTxHandler({ tx: stateTransitionByteArray }) {
+    const { height: blockHeight } = blockExecutionState.getHeader();
+
     const stateTransition = await transactionalUnserializeStateTransition(stateTransitionByteArray);
 
+    logger.debug(`Deliver state transition ${stateTransition.hash()} from block #${blockHeight}`);
```

Comment:
What if an error from `transactionalUnserializeStateTransition` will be thrown? We need to log it higher.